frontend/ui/screensaver.lua

```diff
@@ -429,8 +430,7 @@ function Screensaver:show(event, fallback_message)
 
         -- check if message_widget should be overlaid on another widget
         if message_widget ~= nil then
-            if widget ~= nil
-            then
+            if widget then      -- we have a screensaver widget
```

Comment:
No real need for that much spacing before the comment (I guess it's ok when you want to align it with other comments, but it looks strange here).

Reply:
Sure... I do like a bit of extra whitespace but happy to change it.